### ISSUE TITLE
Errors when viewing trim smithing recipes (1.20.6, Fabric)

### ISSUE THREAD

**Issue by Patbox** [emilyploszaj/emi#532]
Errors when viewing trim smithing recipes, no extra mods outside of EMI and Fabric API.
```[20:32:24] [Render thread/ERROR]: [EMI] Exception serializing stack golden_boots{minecraft:trim=>net.minecraft.class_8053@36c5e133} x1
[20:32:24] [Render thread/INFO]: [STDERR]: java.lang.IllegalStateException: Can't access registry ResourceKey[minecraft:root / minecraft:item]; Can't access registry ResourceKey[minecraft:root / minecraft:item]
[20:32:24] [Render thread/INFO]: [STDERR]: 	at com.mojang.serialization.DataResult$Error.getOrThrow(DataResult.java:275)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at com.mojang.serialization.DataResult.getOrThrow(DataResult.java:79)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.api.stack.serializer.EmiStackSerializer.serialize(EmiStackSerializer.java:81)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.api.stack.serializer.EmiStackSerializer.serialize(EmiStackSerializer.java:20)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.registry.EmiIngredientSerializers.serialize(EmiIngredientSerializers.java:32)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.api.stack.serializer.EmiIngredientSerializer.getSerialized(EmiIngredientSerializer.java:19)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.runtime.EmiFavorites.canFavorite(EmiFavorites.java:75)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.api.widget.SlotWidget.addSlotTooltip(SlotWidget.java:247)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.api.widget.SlotWidget.getTooltip(SlotWidget.java:224)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at dev.emi.emi.screen.RecipeScreen.method_25394(RecipeScreen.java:267)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_437.method_47413(class_437.java:120)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_757.mixinextras$bridge$method_47413$180(class_757.java)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_757.wrapOperation$zml000$fabric-screen-api-v1$onRenderScreen(class_757.java:2101)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_757.method_3192(class_757.java:922)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1523(class_310.java:1320)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1514(class_310.java:888)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:265)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
[20:32:24] [Render thread/INFO]: [STDERR]: 	at org.multimc.EntryPoint.main(EntryPoint.java:34)```

**Comment by emilyploszaj** [comment]
@embeddedt do you know why your serializer impl is crashing here?

**Comment by embeddedt** [comment]
No idea. Shouldn't the item registry always exist by virtue of being built-in, and not datapack-driven? :thinking:

**Comment by emilyploszaj** [comment]
This is resolved in 1.1.9
